master/buildbot/newsfragments/setuptools-is-required.bugfix

```diff
@@ -0,0 +1 @@
+setuptools is now marked as required to install buildbot (non-setuptools install was not tested and broken).

```

Comment:
The point is you can add checks for newsfragments to CI, and you don't need an exception for things that don't need to be in the changelog. And they show up in the diff, so it is easier to remember about them in review and consider whether they do need a newsfragement.